interface/wx/textctrl.h

```diff
@@ -219,6 +219,19 @@ enum wxTextAttrLineSpacing
 };
 
 
+/**
+    Underline types that can be used in wxTextAttr::SetFontUnderline().
+
+    @since 3.1.3
+*/
+enum wxTextAttrUnderlineType
+{
+     wxTEXT_ATTR_UNDERLINE_NONE,
+     wxTEXT_ATTR_UNDERLINE_SOLID,
+     wxTEXT_ATTR_UNDERLINE_DOUBLE,
+     wxTEXT_ATTR_UNDERLINE_WAVE
```

Comment:
Maybe call it `SPECIAL`? Calling it "wave" when it's "dotted" under Mac is IMHO more confusing than just saying that it's an underline type which looks differently from the normal one, e.g. wave under MSW/GTK and dotted under Mac.